lib/src/voip/backend/livekit_backend.dart

```diff
@@ -319,6 +322,15 @@ class LiveKitBackend extends CallBackend {
       data,
       EventTypes.GroupCallMemberEncryptionKeysRequest,
     );
+
+    for (final rp in remoteParticipants) {
+      requestEncrytionKeyPending.addAll({
+        rp: RetryEventModel(
+          timeInterval: Duration(seconds: 2),
+          retryFunction: (_) => requestEncrytionKey(groupCall, [rp]),
+        )
+      });
+    }
```

Comment:
```suggestion
    
    requestEncrytionKeyPending.addAll({
      for (final rp in remoteParticipants)
        rp: RetryEventModel(
          timeInterval: Duration(seconds: 2),
          retryFunction: (_) => requestEncrytionKey(groupCall, [rp]),
        )
    });
```

Although I am not so sure about the dispose semantics in that case and you might want to go for:

```dart
for (final rp in remoteParticipants) {
    requestEncryptionKeyPending[rp] ??= RetryEventModel(
          timeInterval: Duration(seconds: 2),
          retryFunction: (_) => requestEncrytionKey(groupCall, [rp]),
        );
}
```

?

Reply:
Looks basically the same to me, do you have any reasons to pick this over the existing one?

Reply:
skipped the ??= because we don't want to throttle it in the sdk by chance